utils/run-unasync.py

```diff
@@ -75,24 +89,37 @@ def main(check=False):
                 filepaths.append(os.path.join(root, filename))
 
     unasync.unasync_files(filepaths, rules)
-
-    if check:
-        # make sure there are no differences between _sync and _sync_check
-        for source_dir in source_dirs:
+    for dir in source_dirs:
+        output_dir = f"{dir[0]}_sync_check/" if check else dir[1]
+        subprocess.check_call(["black", "--target-version=py38", output_dir])
+        subprocess.check_call(["isort", output_dir])
+        for file in glob("*.py", root_dir=dir[0]):
+            # remove asyncio from sync files
```

Comment:
I have a few doubts about 1/ doing this for all files and not only examples and 2/ using a sed command carefully crafted to work on both Linux and macOS. This seems fragile, but it works, so let's do it.

Reply:
Yes, 100% agree on the fragility. I don't feel doing this for all files is a problem, since what we are doing is the correct thing to do as a general case. I could actually make the regex in the sed a bit smarter and capture the function name that is passed to `asyncio.run()` instead of having it hardcoded. I would think this type of transformation would make sense to add to unasync (although not with sed!).